url.bs

```diff
@@ -3050,15 +3050,15 @@ try {
   // that happened
 }</code></pre>
 
- <p>A <a>base URL</a> is necessary if the input is a <a>relative-URL string</a>:
+ <p>For those cases a <a>base URL</a> is necessary:
 
  <pre><code class="lang-javascript">
 var input = "/🍣🍺",
     url = new URL(input, document.baseURI)
 url.href // "https://url.spec.whatwg.org/%F0%9F%8D%A3%F0%9F%8D%BA"</code></pre>
 
- <p>A {{URL}} object can be used as <a>base URL</a> (while IDL requires a string as argument, a
- {{URL}} object stringifies to its {{URL/href}} getter return value):</p>
+ <p>A {{URL}} object can be used as a <a>base URL</a> (while <cite>Web IDL</cite> requires a string
```

Comment:
I might say "the IDL requires" instead of stating that the _Web IDL_ spec has something to say about this specific constructor.